src/architecture/utilitiesSelfCheck/_Documentation/gaussMarkov/secModelDescription.tex

```diff
@@ -1,3 +1,3 @@
 \section{Model Description}
 
-This utility produces white noise. It can have a bias added to it to make a non-zero mean. It also generates "random walk" by treating the noise as additive. Walk bounds can be provided that push the random walk away from the boundary. This is not a saturate or rail, but rather an exponential correction.
+This utility produces white noise. It can have a bias added to it to make a non-zero mean. It also generates "random walk" by treating the noise as additive. Walk bounds can be provided that push the random walk away from the boundary. Setting the bounds to zero or less disables the random walk, this is the default.
```

Comment:
Be sure to TeX the documentation file and add the resulting PDF.  The TeX'ing is not done by the automated documentation build system.